Extension/CHANGELOG.md

```diff
@@ -2202,4 +2202,4 @@
 ## Version 0.5.0: April 14, 2016
 * Usability and correctness bug fixes.
 * Simplify installation experience.
-* Usability and correctness bug fixes.
\ No newline at end of file
+* Usability and correctness bug fixes.
```

Comment:
I don't see a change here. It seems like this duplicate line should be deleted.

Reply:
Probably the EOF character issue